spec/core_functions/meta/accepts_content.hrx

```diff
@@ -0,0 +1,140 @@
+<===> options.yml
+---
+:todo:
+- sass/libsass#2807
+
+<===>
+================================================================================
+<===> accepts/direct_child/input.scss
+@use "sass:meta";
+
+@mixin a() {@content;}
```

Comment:
I made this change, but I'll mention that I had intentionally left this semicolon because syntax highlighting (at least in vscode) breaks for Sass files -- and hrx files containing Sass -- when `@content` isn't followed by a semicolon.

![image](https://github.com/sass/sass-spec/assets/39542938/dac85954-6548-4516-a118-e70a9e5fd3df)

![image](https://github.com/sass/sass-spec/assets/39542938/c3dfa40e-6a4d-4e6b-aaab-b6e10cad0cf8)

Reply:
That sounds like a bug in the tooling.

Reply:
It is, and it's quite annoying. I know it's bad to change the contents of the spec to workaround the bad tooling, but I assume most people use vscode to edit these files, and in some cases the syntax highlighting gets so bad that it's better to just turn it off. There's a number of different cases like this in vscode for all Sass files,

![image](https://github.com/sass/sass-spec/assets/39542938/39daefc1-a941-44f5-b5d4-c3f55e09efab)
![image](https://github.com/sass/sass-spec/assets/39542938/bde09292-6f84-44a4-8011-732956955c0e)

The syntax highlighting error shown here is pretty minimal, but it demonstrates the bug. `@content` without a semicolon isn't the only similar issue where there's valid Sass syntax and the editor will either red underline as a syntax error or just completely break down while doing syntax highlighting for the entire file.

I started a bit on an LSP implementation for Sass using my compiler and some of rust-analyzer's internals.

Reply:
I wonder if this particular bug could be fixed by changing [this regex](https://github.com/microsoft/vscode/blob/d9c889a0f762cb954875eb29f5c9df2a7df8da30/extensions/scss/syntaxes/scss.tmLanguage.json#L108) to recognize `[;\}]` instead of just `;`.

Reply:
The above change seems to work. https://github.com/microsoft/vscode/pull/194363